main/controller/team_controller.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render, redirect
+
+from main.service import team_service
+from main.model_form.team_form import TeamForm
+from django.views import View
+
+
+class CreateTeamView(View):
+    team_form = TeamForm
+
+    def get(self, request):
+        form = self.team_form()
+        return render(request, "team/create.html", {'form': form})
+
+    def post(self, request):
+        form = self.team_form(request.POST)
+        if form.is_valid():
+            team = team_service.create(form)
```

Comment:
We actually should pass "request.POST" to "team_service.create" instead of passing a Team instance. The "create" method will validate the form data and save to the teams table. If any error occurs in any othe above two steps, "create" method throws appropiraite Exception that the controller catches and provides appropriate HTTP Response.